protocols/luci-proto-openconnect/htdocs/luci-static/resources/protocol/openconnect.js

```diff
@@ -93,6 +93,12 @@ return network.registerProtocol('openconnect', {
 		    certLoadPromise = null,
 		    o;
 
+                o = s.taboption('general', form.ListValue, 'vpn_protocol', _('VPN Protocol'));
```

Comment:
Can you please fix the indentation here? (Tabs vs. spaces)

Reply:
done.